### PR TITLE
Documentation: KBD75

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

I was sent a KBD75 Round 1 PCB for porting from Bootmapper Client to QMK. Backstory is that the user had tried to no avail, the rev1 and rev2 firmwares. After consultation with me, we discovered that the board was not supported in QMK. 

In the middle of matrix tracing, I recognized the switch matrix. It's the same one as the Winkeyless.kr B.mini. 

This PR has no code change and is just an update of the documentation pointing other potential users who own Round 1 or Round 2 of the KBD75 PCB to the correct sources. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mechmerlin** [comment]
Lol yan's firmware size is too large.

File: keyboards/kbdfans/kbd75/readme.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ Keyboard Maintainer: QMK Community
 Hardware Supported: KBD75 PCB rev 1 and 2  
 Hardware Availability: https://kbdfans.myshopify.com/products/kbd75-keyboard-set?variant=35638534029
 
-The KBD75 has two revisions. Revision 2 has a USB C port and allows a numpad layout. `.hex` files made
-for KBD75 revision 1, will still work for revision 2. `.hex` files made for revision 2 (excluding the numpad)
-will also still work on revision 1. Both revisions share the same switch matrix but have switch holes in 
-different areas. 
+The KBD75 was sold through various rounds, with various PCB changes between them. 
+
+* **Round 1 (2017)**: Black PCB - Bootmapper Client, atmega32a MCU (Use Winkeyless.kr B.mini firmware)
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd75/readme.md]:
```suggestion
* **Round 1 (2017)**: Black PCB - Bootmapper Client, atmega32a MCU (Use [Winkeyless.kr B.mini](../../winkeyless/bmini/) firmware)
```

File: keyboards/kbdfans/kbd75/readme.md [hunk 2]
```diff
@@ -4,10 +4,12 @@ Keyboard Maintainer: QMK Community
...
+* **Round 2 (2018)**: White PCB - Bootmapper Client, atmega32a MCU (Use Winkeyless.kr B.mini firmware)
```
Comment (noroadsleft) [inline keyboards/kbdfans/kbd75/readme.md]:
```suggestion
* **Round 2 (2018)**: White PCB - Bootmapper Client, atmega32a MCU (Use [Winkeyless.kr B.mini](../../winkeyless/bmini/) firmware)
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**